doc/Programming-Guide/01_Main.dox

```diff
@@ -7,7 +7,7 @@
  */
 
 /**
-\mainpage Squid 3.x Developer Programming Guide
+\mainpage Squid Developer Programming Guide
```

Comment:
"Developer Programming Guide" is ambiguous and excessive IMO.

```suggestion
\mainpage Squid Programming Guide
```

or


```suggestion
\mainpage Squid Development Guide
```

I do not insist on any changes.